packages/xchain-cosmos/src/client.ts

```diff
@@ -282,9 +294,9 @@ class Client implements CosmosClient, XChainClient {
       this.registerCodecs()
 
       const mainAsset = this.getMainAsset()
-      const txHistory = await this.sdkClient.searchTx({
+      const txHistory = await this.getCosmosClientSdk(0).searchTx({
```

Comment:
Again reinstantiating this is heavy; should we have a "anonClient" instantiated used for this kind of function call?